test/it/uk/gov/hmrc/exports/base/IntegrationTestPurgeSubmissionsToolSpec.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmrc.exports.base
+
+import org.mongodb.scala.MongoCollection
+import org.mongodb.scala.bson.BsonDocument
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import uk.gov.hmrc.exports.repositories.{
+  DeclarationRepository,
+  ParsedNotificationRepository,
+  SubmissionRepository,
+  UnparsedNotificationWorkItemRepository
+}
+
+trait IntegrationTestPurgeSubmissionsToolSpec extends IntegrationTestSpec with GuiceOneAppPerSuite {
```

Comment:
Would be nice if, instead of `IntegrationTestSpec`, you could use `IntegrationTestBaseSpec` which already includes `GuiceOneAppPerSuite` and use `test-customs-declare-exports` as db under test